studio/components/interfaces/Settings/General/CustomDomainConfig/CustomDomainConfig.utils.ts

```diff
@@ -0,0 +1,9 @@
+// [Joshen] Should tally with https://github.com/supabase/cli/blob/63790a1bd43bee06f82c4f510e709925526a4daa/internal/utils/api.go#L98
+export const verifyCNAME = async (domain: string): Promise<boolean> => {
```

Comment:
small nit: you should wrap this function into a `react-query` and turn it into a hook. You're using the same abstractions (`isActivated`, `onSuccess`, `onError`) already.

Reply:
Ohh good point actually - will do that